samples/WebApi/rd.xml

```diff
@@ -0,0 +1,164 @@
+<Directives>
+    <Application>
+        <Assembly Name="SampleWebApi" Dynamic="Required All" />
+        <Assembly Name="System.Private.CoreLib">
+            <Type Name="System.Runtime.CompilerServices.CompilationRelaxationsAttribute" Dynamic="Required All" />
```

Comment:
Did you have a look at reducing the list of things we have here? If not, I can have a look later over the weekend - I doubt we need any of the attribute stuff anymore, and a lot of the other stuff might be removable as well.